cpu/pmqa.py.data/pmqa.yaml

```diff
@@ -0,0 +1,7 @@
+#Usage: run_arg can  any of one from below list
```

Comment:
Can't it be more than one? Like `cpufreq` and `cputopology`? I guess it could, since you're accepting `all`.